tools/depends/target/Toolchain.cmake.in

```diff
@@ -32,23 +32,37 @@ elseif(OS STREQUAL osx)
   set(CORE_SYSTEM_NAME osx)
 elseif(OS STREQUAL ios)
   set(CMAKE_SYSTEM_NAME Darwin)
-  set(CORE_SYSTEM_NAME ios)
+  if(PLATFORM STREQUAL appletvos)
+    set(CORE_SYSTEM_NAME tvos)
```

Comment:
Remove this from the PR as the code which may need it is not added in this PR.

Reply:
Toolchain file removes.
You are right; this file is not needed to build deps.
We keep theses changes for a future PR concerning Kodi core.